integrations/acquisition/covidcast/test_covidcast_meta_caching.py

```diff
@@ -58,6 +58,19 @@ def setUp(self):
     secrets.db.host = 'delphi_database_epidata'
     secrets.db.epi = ('user', 'pass')
 
+    epidata_cnx = mysql.connector.connect(
+        user='user',
+        password='pass',
+        host='delphi_database_epidata',
+        database='epidata')
+    epidata_cur = epidata_cnx.cursor()
+
+    epidata_cur.execute("DELETE FROM `api_user`")
+    epidata_cur.execute('INSERT INTO `api_user`(`api_key`, `email`) VALUES("key", "email")')
+    epidata_cnx.commit()
+    epidata_cur.close()
+    epidata_cnx.close()
```

Comment:
you use this a couple times...  should we make it into a little helper method stored in `src/common/integration_test_base_class.py`?  it doesnt quite fit the requirements of https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming), so its fine to leave as-is.

Reply:
Yeah, I added this in places where we `unittest.TestCase` is used and we don't have connection to the `epidata` database. This can be fixed in next round of refactoring.